keyboards/idobao/montex/v2/keymaps/via/config.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2022 IBNobody & vinorodrigues
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
+
+#if defined(RGB_MATRIX_ENABLE)
+    #define VIA_QMK_RGBLIGHT_ENABLE
+#endif
```

Comment:
This won't be needed anymore, as of the end of the month (or if you compile from the develop branch).
```suggestion
```